src/pp-annotator/viewer/PrzypisViewer.tsx

```diff
@@ -1,21 +1,42 @@
-import React from 'react';
-import ReactDOM from "react-dom";
+import * as React from 'react';
+import * as ReactDOM from "react-dom";
 
-import AnnotationMultipleViewer from './AnnotationMultipleViewer.jsx';
+import AnnotationMultipleViewer from './AnnotationMultipleViewer';
 
-import { ui, util } from 'annotator';
+import * as annotator from 'annotator';
+import IAnnotation from '../i-annotation';
+import IPosition from '../i-position';
 
+const { ui, util } = annotator;
 const { widget: { Widget } } = ui;
 
 const { $ } = util;
 
+interface IPrzypisViewerOptions extends annotator.ui.widget.IWidgetOptions {
+    defaultFields?: boolean;
+    inactivityDelay?: number;
+    activityDelay?: number;
+    permitEdit?: () => boolean;
+    permitDelete?: () => boolean;
+    autoViewHighlights?: Node;
+    onEdit?: (annotation: IAnnotation) => void;
+    onDelete?: (annotation: IAnnotation) => void;
+}
 
 // Public: Creates an element for viewing annotations.
 export default class PrzypisViewer extends Widget {
     static get nameSpace() {
```

Comment:
to chyba nie musi być getterem, wystarczyłoby `static namespace = 'annotator-viewer';` jak sądzę

Reply:
done